ixbrlparse/components/transform.py

```diff
@@ -113,26 +102,10 @@ def get_format(format_: Optional[str]) -> Type[ixbrlFormat]:
 
     format_ = format_.replace("-", "")
 
-    if format_ in ("zerodash", "numdash", "fixedzero"):
-        return ixtZeroDash
-
-    if format_ in ("nocontent", "fixedempty"):
-        return ixtNoContent
-
-    if format_ in ("booleanfalse", "fixedfalse"):
-        return ixtFixedFalse
-
-    if format_ in ("booleantrue", "fixedtrue"):
-        return ixtFixedTrue
-
-    if format_ in ("numdotdecimal", "numcommadot", "numspacedot"):
-        return ixbrlFormat
-
-    if format_ in ("numcomma", "numdotcomma", "numspacecomma", "numcommadecimal"):
-        return ixtNumComma
-
     if format_ in ("numwordsen"):
         return ixtNumWordsEn
+    elif format_:
+        return ixtWrapper
```

Comment:
I think here is where we need to check the format exists in `ixtNamespaceFunctions` before returning - this will mean that the `NotImplementedError` will still be raised